src-ts/lib/work-provider/work-functions/work-factory/work.factory.ts

```diff
@@ -102,13 +102,15 @@ function buildFormDataData(formData: any): any {
 }
 
 function buildFormDataDesign(formData: any): any {
-    const styleInfo: Array<string> = [
-        `Like: ${formData.likedStyles?.value?.join(', ') || ''}`,
-        `Dislike: ${formData.dislikedStyles?.value?.join(', ') || ''}`,
-        `Additional Details: ${formData.stylePreferences?.value || ''}`,
-        `Color Selections: ${formData.colorOption?.value.join(', ') || ''}`,
-        `Specific Colors: ${formData.specificColor?.value || ''}`,
-    ]
+    const styleInfo: {} = {
+        'Like': formData.likedStyles?.value?.join(', '),
+        // Disabling lint error to maintain order for display
+        // tslint:disable-next-line: object-literal-sort-keys
+        'Dislike': formData.dislikedStyles?.value?.join(', '),
```

Comment:
rather than disabling this lint error, can we just define the keys in alpha order? and do we need quotes for the single word keys?

Reply:
i see. i generally wouldn't use a dictionary to persist an ordered list. are we doing this in a lot of places? would it be too much refactor to change the detail object to a list?

Reply:
I replaced the form detail objects with arrays and removed the line to disable the lint error everywhere EXCEPT for this styleInfo object as it was getting to be too messy and complicated when it came to parsing within WorkDetailDetailsPane. In order to display the "Not provided" text for each of these keys I would have had to have an object within an array